Kiosk/Storyboards/Fulfillment.storyboard

```diff
@@ -89,7 +89,7 @@
         <scene sceneID="wG8-BK-Gy6">
             <objects>
                 <navigationController modalPresentationStyle="formSheet" navigationBarHidden="YES" id="w8O-U4-lhE" customClass="FulfillmentNavigationController" customModule="Kiosk" customModuleProvider="target" sceneMemberID="viewController">
-                    <navigationBar key="navigationBar" contentMode="scaleToFill" misplaced="YES" id="PvK-mZ-4Jp">
+                    <navigationBar key="navigationBar" contentMode="scaleToFill" id="PvK-mZ-4Jp">
```

Comment:
I think it's when what it shows in the Storyboard UI isn't what will be in the app because you have changed constraints but not updated the "positioning" (tends to show as a blue border)